src/ex3/cliTasks.js

```diff
@@ -0,0 +1,94 @@
+import chalk from "chalk";
+import { Command } from "commander";
+import { ItemManage_CLI } from "./ItemManager_cli.js";
+
+const program = new Command();
+const itemManager = new ItemManage_CLI();
+
+program
+  .name("cli-tasks")
+  .version("1.0.0")
+  .description("CLI tasks manager for ex3");
+
+program
+  .command("add")
+  .alias("a")
+  .description("Add new task")
+  .argument("<task>", "Task to add")
+  .action(async (taskInput) => {
+    const isCompleted = false;
```

Comment:
this is not a good practice to pass it from here, no need for that.
if every task created is not completed you should define it only in the add task function 

The cli doesn't need to know what is the data structure of a task